index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styel.css" />
+  </head>
+  <body>
+    <div class="nav-container">
+      <div class="wrapper">
+        <nav>
+          <h2 class="logo">Logo</h2>
+          <img  class="image" src="Enabled.svg" alt="" />
+        </nav>
+      </div>
```

Comment:
- [ ] Kindly, make sure that you wrap your navigation section(logo and navigation hamburger) with a `<header>` tag as requested in the project requirement.
- [ ] Also, instead of using `<h2>` tag for your logo, kindly use `<a>` tag for it. you can implement it like this:  `<a href="#" class="logo">Logo</a>`